src/libraries/System.IO.FileSystem/tests/Enumeration/SkipAttributeTests.cs

```diff
@@ -25,21 +25,24 @@ public void SkippingHiddenFiles()
         {
             DirectoryInfo testDirectory = Directory.CreateDirectory(GetTestFilePath());
             DirectoryInfo testSubdirectory = Directory.CreateDirectory(Path.Combine(testDirectory.FullName, GetTestFileName()));
-            FileInfo fileOne = new FileInfo(Path.Combine(testDirectory.FullName, GetTestFileName()));
 
-            // Put a period in front to make it hidden on Unix
-            FileInfo fileTwo = new FileInfo(Path.Combine(testDirectory.FullName, "." + GetTestFileName()));
+            FileInfo fileOne = new FileInfo(Path.Combine(testDirectory.FullName, GetTestFileName()));
             FileInfo fileThree = new FileInfo(Path.Combine(testSubdirectory.FullName, GetTestFileName()));
+
+            // Put a period in front of files two and four to make them hidden on Unix
+            FileInfo fileTwo = new FileInfo(Path.Combine(testDirectory.FullName,     "." + GetTestFileName()));
             FileInfo fileFour = new FileInfo(Path.Combine(testSubdirectory.FullName, "." + GetTestFileName()));
```

Comment:
Are there tests for UserFlags.UF_HIDDEN?

Reply:
This is what I am investigating currently.

Reply:
I added a unit test to verify this flag in OSX, and I debugged it manually to check that it worked.